jupyterhub_moss/form/option_form.js

```diff
@@ -42,7 +42,7 @@ function createEnvironmentDiv(key, description, path, modules, checked = false)
   const label = document.createElement('label');
   label.setAttribute('for', radio_id);
   label.setAttribute('title', title);
-  label.textContent = description;
+  label.textContent = description ? description : [path, modules].filter(v => v).join(" - ");
```

Comment:
If there is no environment name, use concatenate `path` and `modules`

Reply:
This will probably break the formatting of the advanced tab as paths can be very long and non-breakable. But since it's just a fallback solution is definitely better than nothing.